src/components/common/spinner.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+function SvgSpinner() {
```

Comment:
Le's follow the existing naming pattern and rename the component & file to `SpinnerSvg`. Also please place the svg into a folder with `SpinnerSvg` name. Also add an `index.js` file to have proper imports.